blocks/cards/cards.css

```diff
@@ -0,0 +1,157 @@
+
+
+.cmp-cards__inner-wrap{
+    --cards-page-title-font-size: var(--font-size-4000);
+}
+
+.cmp-cards__inner-wrap h1 {
+    color: var(--reactive-text-color);
+    font-size: var(--cards-page-title-font-size);
+    margin: 2rem auto;
+    padding-right: 3.5rem;
+    padding-left: 3.5rem;
+    width: 70%;
+    max-width: var(--layout-max-width-desktop);
+}
+
+.cmp-cards__inner-wrap > h2 { 
+    --cards-page-subhead-font-size: var(--font-size-900);
+
+    color: var(--reactive-text-color);
+    font-size: var(--cards-page-subhead-font-size);
+    font-weight: 100;
+    margin: 2rem auto;
+    padding-right: 3.5rem;
+    padding-left: 3.5rem;
+    width: 70%;
+    max-width: var(--layout-max-width-desktop);
+    margin-bottom: 4em;
+}
+
+.callout{
+    width: var(--content-width);
+    background: var(--color-base-white);
+    margin: auto;
+    max-width: var(--layout-max-width-desktop);
+    color: var(--color-base-red);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.callout>div{
+    font-size: var(--font-size-2500);
+    font-weight: var(--font-weight-black);
+    line-height: 1.2em;
+    margin: 4.2em 15%;
+}
+
+.cards.block {
+    background-color: transparent;
+    display: grid;
+    grid-template-columns: repeat(12, [col-start] 1fr);
```

Comment:
I _think_ this needs to switch to `6` columns until we hit the first (`900px`) breakpoint:

```suggestion
    grid-template-columns: repeat(6, [col-start] 1fr);
```

With this at `12`, and the children at `grid-column: col-start 1 / span 6;`, the children only span half of the grid.

<img width="786" alt="Screen Shot 2022-02-17 at 10 47 38 AM" src="https://user-images.githubusercontent.com/360251/154518370-02d8c68a-9a21-4a18-b88a-51bf13f8b97a.png">